proc/eval.go

```diff
@@ -38,7 +38,8 @@ func (scope *EvalScope) evalAST(t ast.Expr) (*Variable, error) {
 			if err == nil {
 				return v, nil
 			}
-			if err != reader.TypeNotFoundErr {
+			_, isident := node.Fun.(*ast.Ident)
```

Comment:
Can you add a quick comment explaining this assignment, and the usage in the branch.